demos/notebooks/demo_multisession_registration.ipynb

```diff
@@ -118,13 +126,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Combining data of components over multiple sessions\n",
+    "## Combining data of components over multiple sessions (optional)\n",
     "\n",
     "Now that all sessions are aligned and we have a list of re-registered neurons, we can use `assignments` and `matchings` to collect traces from neurons over different sessions.\n",
     "\n",
     "As an exercise, we can collect the traces of all neurons that were registered in all sessions. We already gathered the indices of these neurons in the previous cell in `assignments_filtered`. Assuming that traces of each session are saved in their own `CNMF` object collected in a list, we can iterate through `assignments_filtered` and use these indices to find the re-registered neurons in every session.\n",
     "\n",
-    "Note: This notebook does not include the traces of the extracted neurons, only their spatial components. As such the loop below will produce an error. However, it demonstrates how to use the results of the registration to in your own analysis to extract the traces of the same neurons across different sessions."
+    "Note: This notebook does not include the traces of the extracted neurons, only their spatial components. As such the loop below will produce an error if you uncomment it. However, it demonstrates how to use the results of the registration to in your own analysis to extract the traces of the same neurons across different sessions."
```

Comment:
Would it be better to more deeply adjust the notebook so we don't need to have commented-out code? Or would that impose too much RAM or storage requirements on the notebook?

Reply:
This cell always threw an error -- that code was just demonstrating how you might do something if you had data that we don't provide. Commenting it out is just to have the nb run without error.

Reply:
Should we make adjustments so the data is there to run the cell instead?

Reply:
(based on slack conversation, this is fine for now)